torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4111,8 +4111,26 @@ def shape(size, rank, with_batch_channel=True):
             return tuple([N, C] + ([size] * rank))
         return tuple([size] * rank)
 
-    make_arg = partial(make_tensor, device=device, dtype=dtype,
-                       requires_grad=requires_grad, low=-1, high=1)
+    if mode in ('bilinear', 'bicubic'):
+        make_arg = partial(
+            make_tensor,
+            device=device,
+            dtype=dtype,
+            requires_grad=requires_grad,
+            low=0 if dtype == torch.uint8 else None,
+            high=256 if dtype == torch.uint8 else None,
+        )
+        # provide a single sample for more typical image processing usage
+        rank = 2
+        yield SampleInput(
+            make_arg(shape(400, rank)),
+            shape(270, rank, False),
+            None,
+            mode,
+            False,
```

Comment:
It would be more readable if the keyword variant for these parameters is used.

Reply:
As discussed together with Philip, I'll add keywords everywhere in this function for readability.